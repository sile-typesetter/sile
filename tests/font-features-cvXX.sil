\begin[papersize=a6]{document}
\nofolios
\neverindent
\script[src=packages/features]
\set[parameter=harfbuzz.subshapers,value=ot]
һ ồ

\begin[CharacterVariant="{75,81}"]{font}
һ ồ

\font[CharacterVariant=No81]{һ} \font[CharacterVariant=No75]{ồ}

\font[CharacterVariant="{No75, No81}"]{һ ồ}
\end{font}
\end{document}
