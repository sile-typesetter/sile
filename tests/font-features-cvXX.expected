Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	30.4442
Set font 	Gentium Plus;10;400;;normal;;LTR
T	858 w=5.5176	(һ)
Mx 	23.0701
T	1306 w=5.0293	(ồ)
Mx 	14.8819
My 	42.4442
Set font 	Gentium Plus;10;400;;normal;+cv75;+cv81;LTR
T	859 w=5.2148	(һ)
Mx 	22.7674
T	1307 w=5.0293	(ồ)
Mx 	14.8819
My 	54.4442
Set font 	Gentium Plus;10;400;;normal;+cv75;-cv81;LTR
T	858 w=5.5176	(һ)
Mx 	23.0701
Set font 	Gentium Plus;10;400;;normal;-cv75;+cv81;LTR
T	1306 w=5.0293	(ồ)
Mx 	14.8819
My 	66.4442
Set font 	Gentium Plus;10;400;;normal;-cv75;-cv81;LTR
T	858 w=5.5176	(һ)
Mx 	23.0701
T	1306 w=5.0293	(ồ)
End page
Finish
